site/src/components/GroupAvatar/GroupAvatar.tsx

```diff
@@ -0,0 +1,43 @@
+import Avatar from "@material-ui/core/Avatar"
+import Badge from "@material-ui/core/Badge"
+import { withStyles } from "@material-ui/core/styles"
+import Group from "@material-ui/icons/Group"
+import React from "react"
+import { firstLetter } from "util/firstLetter"
+
+const StyledBadge = withStyles((theme) => ({
+  badge: {
+    backgroundColor: theme.palette.background.paper,
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: "100%",
+    width: 24,
+    height: 24,
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+
+    "& svg": {
+      width: 14,
+      height: 14,
+    },
+  },
+}))(Badge)
```

Comment:
What does `(Badge)` do here? Is that applying the styles for `Badge` as well?

Reply:
Yes, it is a better way, IMO, to modify component styles.